### DISCUSSION TITLE
Any way to get textInput props or any element props?

### DISCUSSION THREAD

**Discussion by johhansantana** [wix/Detox#2943] [Q&A]
I'm trying to check if a textInput has the `secureTextEntry` prop enabled/disabled (boolean) but I'm not sure how to get the value when using the e2e testing.

**Edit**
after further reading I think this is something that would be better to test in unit testing instead of e2e. I'm still very new at doing these automated testings on the front-end so if anyone can confirm this Ill appreciate.

**Comment by lukebrandonfarrell** [comment]
@johhansantana you could make your testID dynamic i.e. `my-text-input-plain`, `my-text-input-secure` and update the test ID when the user makes the input 'secure'. Maybe that will help?

**Comment by d4vidi** [comment]
>better to test in unit testing instead of e2e.

I largely agree
